.buildkite/commands/run-ui-tests.sh

```diff
@@ -27,4 +27,8 @@ install_cocoapods
 echo "--- 🧪 Testing"
 xcrun simctl list >> /dev/null
 rake mocks &
-bundle exec fastlane test_without_building name:"$TEST_NAME" device:"$DEVICE" ios_version:"$IOS_VERSION"
+bundle exec fastlane test_without_building \
+  name:UITests-CI \
+  configuration:"$CONFIGURATION" \
```

Comment:
Is this argument `configuration`, or `test_configuration` that's parsed [here](https://github.com/woocommerce/woocommerce-ios/pull/7415/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R1056)?